.gitignore

```
@@ -0,0 +1,4 @@
+# Compiled source #
+###################
+*.o
+*.exe
```

```suggestion
*.exe
LogInDataBase
```

That would be my suggestion.

---

I just checked and I'm pretty sure that the LogInDataBase and the .dev file were generated by Dev C++. 
The IDE already generated a .exe but i never upload it, I thought (wrongly) that you only uploaded the .exe in the releases, but as I can see now you have to add it to the code too.

The reason why I'm saying those two files are generated by Dev C++ is cause I created a new project in another IDE, copied the .cpp and .o files of this project and compiled and those files weren't generated

---

Got it, then let's remove all Dev C ++ files from this repository as they depend on the IDE.
The `.exe` would be part of a release. You don't have to add it to the repo, that's why we want git to ignore that file.